Makefile

```
@@ -108,7 +108,7 @@ KUBECONFORM_FLAGS := \
 	-schema-location 'https://raw.githubusercontent.com/datreeio/CRDs-catalog/main/{{.Group}}/{{.ResourceKind}}_{{.ResourceAPIVersion}}.json'
 
 .PHONY: test-helm
-test-helm: $(KUBECONFORM) verify-helm-docs ## Test Helm charts (lint, template, validate, kubeconform)
+test-helm: verify-helm-docs ## Test Helm charts (lint, template, validate, kubeconform)
```

checkmake (0.3.2)

[warning] Target body for "test-helm" exceeds allowed length of 5 lines (214).

(maxbodylength)